packaging/fedora/uberftp.spec

```diff
@@ -20,7 +21,8 @@ It supports GSI authentication, parallel data channels and
 third party transfers.
 
 %prep
-%setup -q -n UberFTP-%{gittag}
+#%setup -q -n UberFTP-%{gittag}
+%setup -q -n uberftp-client-2.9-Beta
```

Comment:
see other comment: why the client? And why hardcoded version here?

Reply:
See above.

Reply:
Fixed with https://github.com/gridcf/UberFTP/pull/16/commits/59bdc2b0cd195516a7a8187e295ae9ce555f19fe